src/svea_sensors/launch/localize.launch

```diff
@@ -32,8 +32,12 @@
     <arg name="start_rtk"                 default="false"/>
     <arg name="gps_device"              default="/dev/ttyACM0" />
     <arg name="delay"                   default="2.0"/>
+    <arg name="gps_baud"            default="250000" />  <!-- 38400 for UART connection -->
     <!-- yaw offest = initial_pose_a -->
 
+    <!-- IMU -->
+    <arg name="start_imu"                 default="false"/>
```

Comment:
Should this be false by default?

Reply:
yes, since the imu is connected to the teensy in the new configuration. The imu.launch file is only for the case if it is connected to the Tx2.

Reply:
Okay, then we should have a better name or comment about when/how to use this argument